%YAML:1.0

# # realsensed400
rospackage_path: "/home/wpr/code/catkin_ws/src/orbslam_semantic_nav_ros/"
color_img_topic: "/d400/color/image_raw"
depth_img_topic: "/d400/aligned_depth_to_color/image_raw"
map_frame_id_param: "/map"
camera_frame_id_param: "/camera_link"
name_of_node: "/IM_robot"
map_file_name_param: "config/map.bin"
voc_file_name_param: "Vocabulary/ORBvoc.txt"
settings_file_name_param: "config/realsensed400.yaml"
load_map_param: 0
publish_pointcloud_param: 0
publish_pose_param: 1
min_observations_per_point: 2

# # realsensed435
# rospackage_path: "/home/wpr/code/catkin_ws/src/orbslam_semantic_nav_ros/"
# color_img_topic: "/d400/color/image_raw"
# depth_img_topic: "/d400/aligned_depth_to_color/image_raw"
# map_frame_id_param: "/map"
# camera_frame_id_param: "/camera_link"
# name_of_node: "/IM_robot"
# map_file_name_param: "config/map.bin"
# voc_file_name_param: "Vocabulary/ORBvoc.txt"
# settings_file_name_param: "config/realsensed400.yaml"
# load_map_param: 0
# publish_pointcloud_param: 0
# publish_pose_param: 1
# min_observations_per_point: 2

# kinect
# rospackage_path: "/home/wpr/code/catkin_ws/src/orbslam_semantic_nav_ros/"
# color_img_topic: "/kinect2/qhd/image_color_rect"
# depth_img_topic: "/kinect2/qhd/image_depth_rect"
# map_frame_id_param: "/map"
# camera_frame_id_param: "kinect2_rgb_optical_frame"
# name_of_node: "/IM_robot"
# map_file_name_param: "config/map.bin"
# voc_file_name_param: "Vocabulary/ORBvoc.txt"
# settings_file_name_param: "config/kinectv2.yaml"
# folder_path: "/home/wpr/code/catkin_ws/src/orbslam_semantic_nav_ros/data/"
# load_map_param: 0
# publish_pointcloud_param: 1
# publish_pose_param: 1
# min_observations_per_point: 2
